packages/@sfcc-bff/productapi/src/api/schema/productDetailsResolvers.js

```diff
@@ -7,37 +7,20 @@
 'use strict';
 
 import Product from '../models/Product';
+import utilities from '../helpers/utils.js';
 import CommerceSdk from 'commerce-sdk';
```

Comment:
Again, most if not all changes here are already in integration branch.  Maybe rebase is needed?

Reply:
same as above